data/mods/Xedra_Evolved/mutations/paraclesians/homullus_mutations.json

```
@@ -0,0 +1,292 @@
+[
+  {
+    "type": "mutation",
+    "id": "HOMULLUS_SKIN_1",
+    "name": { "str": "Mannequin Skin" },
+    "points": 0,
+    "visibility": 2,
+    "ugliness": 0,
+    "description": "Your skin has turned the color of antique dolls.  Your skin is now a pearlescent white.  This has no gameplay effects.",
+    "types": [ "skin_tone" ],
+    "leads_to": [ "HOMULLUS_SKIN_2" ],
+    "category": [ "HOMULLUS" ]
+  },
+  {
+    "type": "mutation",
+    "id": "HOMULLUS_SKIN_2",
+    "name": { "str": "Porcelain Skin" },
+    "points": 2,
+    "visibility": 10,
+    "ugliness": -3,
+    "description": "Your skin has smoothed and all blemishes fade quickly away.  Somewhat reduces wet effects and is tougher but heals a little slower.",
+    "types": [ "SKIN" ],
+    "category": [ "HOMULLUS" ],
+    "prereqs": [ "HOMULLUS_SKIN_1" ],
+    "changes_to": [ "HOMULLUS_SKIN_3" ],
+    "healing_multiplier": 0.9,
+    "wet_protection": [
+      { "part": "head", "ignored": 2 },
+      { "part": "leg_l", "ignored": 5 },
+      { "part": "leg_r", "ignored": 5 },
+      { "part": "foot_l", "ignored": 2 },
+      { "part": "foot_r", "ignored": 2 },
+      { "part": "arm_l", "ignored": 4 },
+      { "part": "arm_r", "ignored": 4 },
+      { "part": "hand_l", "ignored": 1 },
+      { "part": "hand_r", "ignored": 1 },
+      { "part": "torso", "ignored": 10 }
+    ],
+    "integrated_armor": [ "integrated_porcelain_skin" ]
+  },
+  {
+    "type": "mutation",
+    "id": "HOMULLUS_SKIN_3",
+    "name": { "str": "Latex Skin" },
+    "points": 3,
+    "visibility": 10,
+    "ugliness": -6,
+    "mixed_effect": true,
+    "description": "Your skin has become latex-like.  It provides some additional protection and attractiveness but heals at a much slower rate.",
+    "types": [ "SKIN" ],
+    "prereqs": [ "HOMULLUS_SKIN_2" ],
+    "threshreq": [ "THRESH_HOMULLUS" ],
+    "category": [ "HOMULLUS" ],
+    "healing_multiplier": 0.7,
+    "wet_protection": [
+      { "part": "head", "ignored": 5 },
+      { "part": "leg_l", "ignored": 16 },
+      { "part": "leg_r", "ignored": 16 },
+      { "part": "foot_l", "ignored": 5 },
+      { "part": "foot_r", "ignored": 5 },
+      { "part": "arm_l", "ignored": 14 },
+      { "part": "arm_r", "ignored": 14 },
+      { "part": "hand_l", "ignored": 4 },
+      { "part": "hand_r", "ignored": 4 },
+      { "part": "torso", "ignored": 30 }
+    ],
+    "integrated_armor": [ "integrated_latex_skin" ]
+  },
+  {
+    "type": "mutation",
+    "id": "SUMMON_FERAL",
+    "name": { "str": "Summon the lost and the damned" },
```

```suggestion
    "name": { "str": "Lost and the Damned" },
```
Seems a bit more concise of a name.